src/main/java/pl/mantiscrab/budgetr/registration/UserController.java

```diff
@@ -25,6 +24,14 @@ ResponseEntity<UserDto> register(@Valid @RequestBody UserRegisterDto userRegiste
         return ResponseEntity.ok().body(userDto);
     }
 
+    //    TODO: replace this endpoint in tests with one related
+    //    TODO: to business logic and remove
+    @GetMapping("/username")
```

Comment:
User controller not restful :dagger: 

POST /user
PUT /user
PATCH /user
DELETE /user
GET /user